Intro to Python Notebook #1 .ipynb

```diff
@@ -0,0 +1,1718 @@
+    "* variables\n",
+    "\n",
+    "TODO: add exercises\n",
+    "\n",
+    "\n",
+    "References: *Python Data Science Handbook*, Jake VanderPlas. *Effective Computation in Physics*, Anthony Scopatz & Katheryn D. Huff "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 0: Installing Python and Jupyter \n",
+    "\n",
+    "This notebook can be run on your laptop, but you will need python and Jupyter. \n",
+    "The recommended way to setup python is with Anaconda.  Installation instructions for different operating systems can be found here: https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html \n",
+    "\n",
+    "You can then launch Jupyter by typing:\n",
+    "<br>\n",
+    "jupyter notebook \n",
+    "\n",
+    "TODO: instructions for setting up python / jupyter notebook on the cosmology machines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 1:  Running Python\n",
+    "\n",
+    "You're now ready to get started with python! But what exactly is python? Python is a **programming language** that is general-purpose, and high-level. It is extremeley popular in science, engineering and big data fields.  It is designed to be accessible and fun to use - in fact the name comes from the popular British comedy group Monty Python.  Python works as an **interpreter** and translates python **source code** into instructions for your computer's processor. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For example, we can send the instructions to print a message with the print command, to run this cell, click on it, and hit shift+enter this tells jupyter to execute the command. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Go Duke Devils go!\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Go Duke Devils go!')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is in a jupyter notebook. However, python can also be written in text files. If you run the command \"python\" on a file whose name ends in *.py* then Python will execute all of the code in the file. You can try this out by opening up a terminal, cd'ing to this directory and typing \"python hello_world.py\" "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 2: Variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Typing phrases is fun, but our ultimate goal is to be able to use python for science and computation. Just as we did in computation by hand, we can start by defining variables. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Variables can be assigned and name and a value with the \"=\" sign: "
```

Comment:
Probably the _and_ is a typo. Probably good to mention that this is formally an _assignement statement_.